cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -105,7 +109,7 @@ func NewControllerManagerCommand() *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use: "kube-controller-manager",
+		Use: kubeControllerManager,
```

Comment:
It should be better to display the string explicitly here to demonstrate the command usage.